src/PaymentPart/Output/PrintOptions.php

```diff
@@ -2,28 +2,46 @@
 
 namespace Sprain\SwissQrBill\PaymentPart\Output;
 
-enum LineStyle
+/**
+ * @internal This class is meant to be an ENUM. But because it must be compatible
+ * with PHP 8.0, ENUMs should not be used, since they exist only from PHP 8.1
+ * The idea here is to have a usage as close as an ENUM.
+ *
+ * There would be two breaking changes for consumers:
+ * 1. remove the $ in front of the name: LineStyle::$SOLID becomes LineStyle::SOLID.
+ * 2. only an ENUM value could be given to the method setXXX(). Now, standard strings are also accepted
+ */
+final class LineStyle
 {
-    case SOLID;
-    case DASHED;
-    case NONE;
+    public static $SOLID = 'SOLID';
+    public static $DASHED = 'DASHED';
+    public static $NONE = 'NONE';
```

Comment:
You could use `const`, then no `$` is needed.

Reply:
@sprain PHP 8.0 went EOL (no security fixes) 1 ago year. Do we really need to implement backward facing implementations like this to maintain compatibility with it?

Reply:
lol, I just forgot this, thanks :D But that's true that PHP 8.0 is old. We may drop the support. But it means a breaking change...

Reply:
@kohlerdominik 
PHP 8.0 [is only 5% of current installs](https://packagist.org/packages/sprain/swiss-qr-bill/php-stats#all). We can consider dropping it. In a future release, since this is already PHP 8.0 compatible now ;) 

@tafel 
Thanks!
FYI: dropping a PHP version [is not a breaking change](https://github.com/sprain/php-swiss-qr-bill/releases/tag/v3.3).

Reply:
@sprain ok, so composer manage matches between PHP version and packages version itself. The question is, in the current implementation, we can use the `::TOP` const property, but we can also just write `'TOP'` as a string. When/If we migrate the code to enums, it will be a breaking change, because the method will only accept enum type.

Should we go with the current implementation, and document this breaking change later? Or is it better to just drop PHP 8.0 right now, and not worry about breaking change at all?